man/mkdocs/requirements.txt

```
@@ -0,0 +1,4 @@
+mkdocs
+pymdown-extensions
+mkdocs-glightbox
+mkdocs-material
```

Add pyyaml-env-tag and sort:

```suggestion
mkdocs
mkdocs-glightbox
mkdocs-material
pymdown-extensions
pyyaml-env-tag
```